esp/platform/espp.cpp

```diff
@@ -436,13 +436,13 @@ int init_main(int argc, char* argv[])
     catch(IException* e)
     {
         StringBuffer description;
-        ERRLOG("ESP Unhandled IException (%d -- %s)", e->errorCode(), e->errorMessage(description).str());
+        IERRLOG("ESP Unhandled IException (%d -- %s)", e->errorCode(), e->errorMessage(description).str());
```

Comment:
OERRLOG might be better, because the exceptions here are normally config related

Reply:
@mayx Please see previous message

Reply:
(@mayx The wording in the messages refers to "IException", which only makes sense to platform developers. If the message is for Operators, we'd need to re-word the error message so that operators can make sense of the message without refering to the platform code.)